components/onboarding_explore_tools_tour/onboarding_explore_tools_tour_tip.tsx

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import {Placement} from 'tippy.js';
+
+import TourTip from 'components/widgets/tour_tip/tour_tip';
+import {TourTipOverlayPunchOut} from 'components/widgets/tour_tip/tour_tip_backdrop';
+
+import {getLastStep} from '../onboarding_tour/utils';
+
+import useBoardingTourTipManager from './explore_tools_tour_manager';
+
+type Props = {
+    screen: JSX.Element;
+    title: JSX.Element;
+    imageURL?: string;
+    overlayPunchOut: TourTipOverlayPunchOut | null;
+    singleTip?: boolean;
+    placement?: Placement;
+    pulsatingDotPlacement?: Omit<Placement, 'auto'| 'auto-end'>;
+    pulsatingDotTranslate?: {x: number; y: number};
+    offset?: [number, number];
+    width?: string | number;
+}
+
+const OnboardingExploreToolsTourTip = ({
+    title,
+    screen,
+    imageURL,
+    overlayPunchOut,
+    singleTip,
+    pulsatingDotTranslate,
+    pulsatingDotPlacement,
+    offset = [-18, 4],
+    placement = 'right-start',
+    width = 320,
```

Comment:
Since both uses of this are width 352, can we remove `width` as a prop and hard code 352 where it is passed into TourTip down below?

Reply:
Ditto for `placement`, `pulsatingDotPlacement`, and `pulsatingDotTranslate`